support/ppTiming.cpp

```diff
@@ -36,22 +37,22 @@ namespace pumipic {
 
   void SetTimingVerbosity(int v) {
     if (time_per_op.size() > 0) {
-      fprintf(stderr, "[ERROR] Cannot change timing verbosity after first call to RecordTime\n");
+      pPrintError( "[ERROR] Cannot change timing verbosity after first call to RecordTime\n");
```

Comment:
I notice a lot of the `PrintError` messages include [ERROR]`. Would it make sense to include this into the `PrintError` function itself to identify it's an error message?

Reply:
Good Idea! I made that change.